src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.scss

```diff
@@ -106,3 +110,58 @@ $second-color-font: #839C94;
 .star {
   background: url("../../../../../../../assets/img/icon/star-circle.png") no-repeat center;
```

Comment:
Improving long relative paths import:
here and below absolute path to "assets/img/" looks better then "../../.."

Reply:
done